docs/metadata.md

```diff
@@ -0,0 +1,14 @@
+# Volumetric metadata about the VDS
+
+Retrieve basic information related to the VDS file such as axis definitions, overall shape and size, coordinate system, import time stamp and source filename. 
+
+## Response
+On success (200) the response containing the VDS metadata.
```

Comment:
*Content-Type: application/json*
On success (200) the json response contains metdata. See the Metadata model.

?